perf/stability/canary-upgrader/values.yaml

```diff
@@ -1,3 +1,3 @@
 namespace: canary-upgrader
 redeployDays: 2
-kubectlImage: gcr.io/istio-testing/kubectl:latest
+kubectlImage: docker.io/bitnami/kubectl:latest
```

Comment:
Do we want to continue to use the gcr.io repository? Rate limiting may have an effect if using docker.io post Nov 1.

Reply:
would recommend not using `:latest`. I think docker.io is possibly fine though this will only deploy a few times

Reply:
ok updated to 1.18 specifically. And yes this image pull would only happen a few times